clang/include/clang/Basic/Builtins.def

```diff
@@ -33,6 +33,7 @@
 //  q -> Scalable vector, followed by the number of elements and the base type.
 //  Q -> target builtin type, followed by a character to distinguish the builtin type
 //    Qa -> AArch64 svcount_t builtin type.
+//    Qb -> AMDGPU __amdgpu_buffer_rsrc_t builtin type.
```

Comment:
This belongs in the parent patch

Reply:
Well not really. We can't test it out in the parent patch w/o introduction of a builtin function.

Reply:
True it's not testable, but it still logically belongs with the definition of the type

Reply:
It indeed sounds like that, but I checked where `Qa` was introduced. It was after the type was introduced. https://reviews.llvm.org/D150953